README.md

```diff
@@ -1,3 +1,15 @@
 # Download service
 
-Backend server for [https://github.com/cta-epfl/ctadata](https://github.com/cta-epfl/ctadata).
\ No newline at end of file
+Backend server for [https://github.com/cta-epfl/ctadata](https://github.com/cta-epfl/ctadata).
+
+## Installation
+
+For use as a JupyterHub service, run:
+```
+poetry install
+```
+
+For standalone service without JupyterHub :
+```
+poetry install --with dev --without jupyterhub
```

Comment:
seems like I can not make a suggestion that includes ``` since it's used to limit suggestion 🤦. Could you add it?

Reply:
Done 👍